Sming/Components/AxtlsSsl/src/CertificateImpl.h

```diff
@@ -4,34 +4,38 @@
  * http://github.com/SmingHub/Sming
  * All files of the Sming Core are provided under the LGPL v3 license.
  *
- * SslCertificateImpl.h
+ * CertificateImpl.h
  *
  * @author: 2019 - Slavey Karadzhov <slav@attachix.com>
  *
  ****/
 
 #pragma once
 
-#include <Network/Ssl/SslCertificate.h>
+#include <Network/Ssl/Certificate.h>
 #include <axtls-8266/compat/lwipr_compat.h>
 
-class SslCertificateImpl : public SslCertificate
+namespace Ssl
```

Comment:
With the namespacing of one component we should start thinking about introducing namespaces to core and all Sming components.